app/controllers/contracts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ContractsController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def index
+    contracts = []
+    Contract.all.each do |contract|
+      contracts.push(format_contract(contract))
+    end
+    render json: contracts
```

Comment:
`Contract.all.includes(:offer).map { |c| format_contract(c) }` would be more succinct.